src/Serilog/Capturing/PropertyValueConverter.cs

```diff
@@ -214,6 +214,29 @@ bool TryConvertEnumerable(object value, Type type, Destructuring destructuring,
                 {
                     result = SequenceValue.Empty;
                 }
+                else if (list is Array a && a.Rank > 1)
```

Comment:
Perhaps this could be plugged in as an `IScalarConversionPolicy` instead? Since those are checked before this code is reached, we could avoid the extra condition here. We do this already to specialize conversion of byte arrays.

Reply:
`IScalarConversionPolicy` has no access to
1. _depthLimiter (can be solved)
2. `destructuring` boolean argument

Reply:
In my opinion, we should not use `IScalarConversionPolicy` here, because the determining factor whether or not using `IScalarConversionPolicy` is the type of object inside the array, regardless of the dimensions of the array.